packages/devtools_app/lib/src/screens/logging/logging_screen_v2/logging_model.dart

```diff
@@ -98,25 +195,272 @@ class LoggingTableModel extends ChangeNotifier with DisposerMixin {
+  }
+
+  void _handleDeveloperLogEvent(Event e) {
+    final service = serviceConnection.serviceManager.service;
+
+    final logRecord = _LogRecord(e.json!['logRecord']);
+
+    String? loggerName =
+        _valueAsString(InstanceRef.parse(logRecord.loggerName));
+    if (loggerName == null || loggerName.isEmpty) {
+      loggerName = 'log';
+    }
+    final level = logRecord.level;
+    final messageRef = InstanceRef.parse(logRecord.message)!;
+    String? summary = _valueAsString(messageRef);
+    if (messageRef.valueAsStringIsTruncated == true) {
+      summary = '${summary!}...';
+    }
+    final error = InstanceRef.parse(logRecord.error);
+    final stackTrace = InstanceRef.parse(logRecord.stackTrace);
+
+    final details = summary;
+    Future<String> Function()? detailsComputer;
+
+    // If the message string was truncated by the VM, or the error object or
+    // stackTrace objects were non-null, we need to ask the VM for more
+    // information in order to render the log entry. We do this asynchronously
+    // on-demand using the `detailsComputer` Future.
+    if (messageRef.valueAsStringIsTruncated == true ||
+        _isNotNull(error) ||
+        _isNotNull(stackTrace)) {
+      detailsComputer = () async {
+        // Get the full string value of the message.
+        String result =
+            await _retrieveFullStringValue(service, e.isolate!, messageRef);
+
+        // Get information about the error object. Some users of the
+        // dart:developer log call may pass a data payload in the `error`
+        // field, encoded as a json encoded string, so handle that case.
+        if (_isNotNull(error)) {
+          if (error!.valueAsString != null) {
+            final errorString =
+                await _retrieveFullStringValue(service, e.isolate!, error);
+            result += '\n\n$errorString';
+          } else {
+            // Call `toString()` on the error object and display that.
+            final toStringResult = await service!.invoke(
+              e.isolate!.id!,
+              error.id!,
+              'toString',
+              <String>[],
+              disableBreakpoints: true,
+            );
+
+            if (toStringResult is ErrorRef) {
+              final errorString = _valueAsString(error);
+              result += '\n\n$errorString';
+            } else if (toStringResult is InstanceRef) {
+              final str = await _retrieveFullStringValue(
+                service,
+                e.isolate!,
+                toStringResult,
+              );
+              result += '\n\n$str';
+            }
+          }
+        }
+
+        // Get info about the stackTrace object.
+        if (_isNotNull(stackTrace)) {
+          result += '\n\n${_valueAsString(stackTrace)}';
+        }
+
+        return result;
+      };
+    }
+
+    const severeIssue = 1000;
+    final isError = level != null && level >= severeIssue ? true : false;
```

Comment:
`final isError = level != null && level >= severeIssue;` does the same thing.

Reply:
Oops I accidentally included these changes here, but they were supposed to be on the next PR. Removing them.